silkworm/node/huffman/decompressor_test.cpp

```diff
@@ -151,8 +152,8 @@ TEST_CASE("PatternTable::build_condensed", "[silkworm][snapshot][decompressor]")
     std::vector<Pattern> patterns2{{1, v1}, {2, v2}};
     std::map<std::string, std::span<Pattern>> test_spans{
         {"zero patterns", patterns0},
-        {"one pattern", patterns1},
-        {"two patterns", patterns2},
+        {"one pattern", std::span<Pattern>{&patterns1[0], patterns1.size()}},
+        {"two patterns", std::span<Pattern>{&patterns2[0], patterns2.size()}},
```

Comment:
I suggest using `patterns.data()` instead of `&patterns[0]` for improved readability

Reply:
👌